test/evergreen.yml

```diff
@@ -1586,6 +1586,26 @@ tasks:
           display_name: Coverage report
           remote_file: wiredtiger/${build_variant}/${revision}/coverage_report/coverage_report_${build_id}.html
 
+  - name: spinlock-test
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          posix_configure_flags: --enable-python --with-spinlock=gcc --enable-strict
+      - func: "make check all"
+      - func: "test format"
+        vars:
+          times: 3
+      - func: "unit test"
+      - func: "compile wiredtiger"
```

Comment:
Can we separate it into two tests so we can run them in parallel.

Reply:
Good idea! I can break them into two tasks and have a common tag so we can still manage them by single call.

Reply:
If breaking it out into 2 tasks I'd suggest task name `spinlock-gcc-test` and `spinlock-pthread_adaptive-test`.
And, a new tag feels a bit overkill to me :)